src/main/battlecode/server/Config.java

```diff
@@ -46,6 +46,7 @@
         defaults.setProperty("bc.engine.silence-d", "false");
         defaults.setProperty("bc.engine.upkeep", "true");
         defaults.setProperty("bc.engine.breakpoints", "false");
+        defaults.setProperty("bc.engine.debug-methods", "true");
```

Comment:
Can we make it false by default? So we don't accidentally run tournaments with debug methods enabled, and so competitors don't accidentally use methods with debug_ in them (as normal methods) and get a different behavior when they submit online.